client/js/components/assessment_results/_assessment_result.jsx

```diff
@@ -5,23 +5,21 @@ import { connect }                from "react-redux";
 import * as AssessmentActions     from "../../actions/assessment";
 import FormativeResult            from "./formative_result.jsx";
 import SummativeResult            from "./summative_result.jsx";
-import ResultStyles               from "./result_styles.js";
 import CommunicationHandler       from "../../utils/communication_handler";
-import { items }                  from "../../selectors/assessment";
+import { questions }                  from "../../selectors/assessment";
 
 const select = (state) => {
   return {
-    assessmentResult : state.assessment.assessmentResult(),
-    timeSpent        : state.assessment.timeSpent(),
-    questions        : item(state),
-    outcomes         : state.assessment.outcomes(),
-    assessment       : state.assessment.current(),
+    assessmentResult : state.assessmentResult,
```

Comment:
I think assessmentResult and timeSpent should be part of a "user_assessment" portion of the state. Let's discuss the name